docs/REST API/rest-api.md

```diff
@@ -6020,6 +6093,7 @@ Team policies work the same as policies, but at the team level.
 | ------------------ | ------- | ---- | ------------------------------------------------------------------------------------------------------------- |
 | team_id                 | integer | path  | **Required.** Defines what team ID to operate on
 | query                 | string | query | Search query keywords. Searchable fields include `name`. |
+| include_inherited     | boolean | query | If `true`, will include inherited ("All teams") policies in the count. |
```

Comment:
@rachaelshaw and I had some conversations about using `merge_inherited` as the API key for the query and policy counts to keep the naming consistent with the API key for the query and policy list.

https://fleetdm.slack.com/archives/C01EZVBHFHU/p1713889718894049?thread_ts=1713887183.358959&cid=C01EZVBHFHU

Would love for other's input, my concern voiced with using two different keys (`include_inherited` for the counts AND `merge_inherited` for the lists) is that the API will live for a long time and it might feel confusing why it's called `merged_inherited` vs. `include_inherited` unless the context that the list API had separate inherited arrays before but now it is _merged_ vs. the count APIs only counted the team lists but now can _include_ the inherited list in the count.

@jacobshandling @mostlikelee @noahtalerman -- what do y'all think?

Reply:
agree.  i vote we use the same query param on both APIs for consistency/ease.

I personally prefer `include` over `merged`, because the later almost sounds like we're merging data from 2 policies into 1.  `append` could be another option, but I think we use `include` elsewhere in our API.

Reply:
The problem with "include" is that the endpoint to list team policies already includes inherited policies... just in a separate list. So "merge" makes more sense there, because they're always included, you're just specifying _where_. Was thinking if we make the parameters the same, "merge" is the more flexible word for that reason.